app-services/secrets/res/configuration.toml

```diff
@@ -57,10 +57,11 @@ Timeout = "30s"
   Port = 8200
   Path = '/v1/secret/edgex/appservice'
   Protocol = 'http'
+  Tokenfile = 'res/token.json'
 
   [SecretStore.Authentication]
   AuthType = 'X-Vault-Token'
-  AuthToken = 's.BTi2Eb23GWsWtQm9L6r95U5l'
+  AuthToken = 's.sbbMxFh0sNmM7E1v2S0sQ7JD'
```

Comment:
Can you add a comment above this line explaining this token a little.  and maybe just say 'VAULT TOKEN' so people understand this will need to be replaced.